Bundle/CoreBundle/doc/ManageViews.md

```diff
@@ -0,0 +1,31 @@
+#Manage Views
```

Comment:
@Charlie-Lucas 
Can you add a link somewhere to index this file ?
Could be in the CoreBundle/README.md ?